products/BellHybrid/apps/application-bell-settings/presenter/FrontlightPresenter.cpp

```diff
@@ -16,6 +16,11 @@ namespace app::bell_settings
         };
     }
 
+    FrontlightPresenter::~FrontlightPresenter()
+    {
+        exitWithoutSave();
+    }
+
```

Comment:
Are you sure this will work? Imho, the destructor will be invoked even if a user performed full procedure(thus accepted to save the new value) and set the just set value to the default(old) one.

Reply:
you are right! check now